recommended.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable */
+module.exports = {
+  root: true,
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    project: "./tsconfig.json",
+    // sourceType: "module"
+  },
+  plugins: ["@typescript-eslint", "import", "functional", "sonarjs"],
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:sonarjs/recommended",
+    "prettier",
+    "prettier/@typescript-eslint",
+    "plugin:prettier/recommended",
+  ],
+  rules: {
+    "no-case-declarations": "off",
+    "no-inner-declarations": "off",
+    "prefer-const": "error",
+    curly: "error",
+    "spaced-comment": ["error", "always", { block: { balanced: true } }],
+    radix: "error",
+    "one-var": ["error", "never"],
+    "object-shorthand": "error",
+    "no-var": "error",
+    "no-param-reassign": "error",
+    "no-underscore-dangle": "error",
```

Comment:
I'm not sure if it would break our convention of prefixing unused params with underscore. In case, we can add
```suggestion
    "no-underscore-dangle": ["error", { "allowFunctionParams": true }],
```

See https://eslint.org/docs/rules/no-underscore-dangle#options

Reply:
It is, `"allowFunctionParams": true (default) allows dangling underscores in function parameter names`

Reply:
So we want it or not? I think we want to be able to write
```ts
const fn = (_p) => {}
```

Reply:
Sorry, maybe my comment wasn't clear. We can write `const fn = (_p) => {}`, the option `allowFunctionParams` has `true` as default value.